src/test/java/hw2/ComputerTest.java

```diff
@@ -0,0 +1,34 @@
+package hw2;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class ComputerTest {
+    Computer com1 = new Computer("Hp", 15);
```

Comment:
Please convert these to the local variables (move each one to the test method and name just computer) - we do not use them between all methods